mags/RPS.py

```diff
@@ -0,0 +1,151 @@
+# Rock paper scissors game
+import random
+from os import system, name
+from time import sleep
+
+# TODO: rpsls
+
+
+
+# -1, lose; 0, tie; 1, win
+class Player:
+    wins = 0
+    losses = 0
+    ties = 0
+    game_chosen = "RPS"
+    hand_choice = None
+
+    def assign_wins(self, score_int):
+        if score_int == -1:
+            print("Player loses!")
+            self.losses += 1
+        elif score_int == 1:
+            print("Player wins!")
+            self.wins += 1
+        else:
+            self.ties += 1
+
+    def get_score(self):
+        print(f"Player score:\nWins: {self.wins}\nLosses: {self.losses}\nTies: {self.ties}")
+
+# For storing all of the game logic and methods
+class RPS:
+    name = "Rock, Paper, Scissors"
+    choices = ["rock", "paper", "scissors"]
+    win = [0, -1, 1], [1, 0, -1], [-1, 1, 0]
+    outcomes = ["rock smashes scissors!", "paper covers rock!", "scissors cut paper!", "Tie!"]
+
+    def __init__(self, player_choice, computer_choice):
+        # get attack statement
+        # print(f"player choice: {player_choice}. computer choice {computer_choice} comparison matrix: {self.choices.index(player_choice), self.choices.index(computer_choice)}")
```

Comment:
did you intend to leave coding comments?

Reply:
Nope, good catch!